docs/plugins/bootstrap-vue.js

```diff
@@ -1,4 +1,7 @@
 import Vue from 'vue'
 import BootstrapVue from '../../src'
 
-Vue.use(BootstrapVue)
+Vue.use(BootstrapVue, {
+  // Make sure modals in the docs are above anything else
+  BModal: { zIndexOffset: 2000 }
+})
```

Comment:
Any reason why we don't use the new default in the docs?

Reply:
If the new default doesn't work in the docs, then it may not work in production for others.